api/spec/v1.yaml

```diff
@@ -101,6 +101,26 @@ x-common-types:
     - beacon.PVSSCommit
     - beacon.PVSSReveal
     - beacon.VRFProve
+  consensus-event-types: &consensus_event_types
+    - AddEscrow
+    - AllowanceChange
+    - Burn
+    - DebondingStart
+    - DiscrepancyDetected
+    - Entity
+    - ExecutorCommitted
+    - Finalized
+    - Node
+    - NodeUnfrozen
+    - ProposalExecuted
+    - ProposalFinalized
+    - ProposalSubmitted
+    - ReclaimEscrow
+    - Runtime
+    - TakeEscrow
+    - Transfer
+    - Vote
+    - Unknown
```

Comment:
Were these event types taken from [/analyzer/api.go](https://github.com/oasisprotocol/oasis-indexer/blob/main/analyzer/api.go#L129)? Wondering if there's an official list of consensus event types from our documentation. 

Some observations from comparing the indexer event types (`/analyzer/api.go`) to the ones I could find from [oasis-core](https://pkg.go.dev/github.com/oasisprotocol/oasis-core/go/consensus/api#ClientBackend):

- The DiscrepancyDetected event is named `ExecutionDiscrepancyDetected` ([ref](https://pkg.go.dev/github.com/oasisprotocol/oasis-core/go@v0.2202.3/roothash/api#ExecutionDiscrepancyDetectedEvent)). Assuming the oasis-core docs are the ground truth and since we expose event types in the `type` param in the indexer events endpoint, what do you think of renaming this field to `ExecutionDiscrepancyDetected` for consistency here, and we can rename the field in a follow-up PR?
- (minor) The `roothash.InMsgProcessed` event ([ref](https://pkg.go.dev/github.com/oasisprotocol/oasis-core/go@v0.2202.3/roothash/api#InMsgProcessedEvent)) isn't indexed; I assume it was unnecessary when Nikhil first crafted the indexer event types, but I just wanted to confirm that this is still true now that we're adding support for Emerald/Sapphire.
- It seems like we're not using the `Unknown` event type currently (prod db has 0 events with that type). What do you think of removing `Unknown` from the openapi spec? I can do more digging to see if we can fully remove it from the indexer codebase in a follow-up PR too.